### ISSUE TITLE
Hundreds of extra empty divs in .vjs-control-text

### ISSUE THREAD

**Issue by Albii** [videojs/videojs-youtube#358]
Hello and thank you for your amazing job.

I have a problematic issue you can reproduce on the simple example : while buffering any youtube video hundreds of empty divs are added here :

```
<div class="vjs-load-progress">
  <div style="left: 0%; width: 100%;"></div>
  <div style="left: 0%; width: 100%;"></div>
  <div style="left: 0%; width: 100%;"></div>
  <div style="left: 0%; width: 100%;"></div>
  <div style="left: 0%; width: 100%;"></div>
</div>
```

Hundreds of div are appened to this div as buffering goes and it drastically slows down the browser causing small freezes and performances issues.

Any idea ?

**Comment by eXon** [comment]
No idea why this is happening. Do you have the same issue with barebone videojs?

**Comment by Albii** [comment]
No problem with barebone videojs, you can reproduce this easily by checking it on your examples/simple.html file. I guess this has something to do with an event getting triggered on buffering that causes videojs to append this empty div (which is used to display the load progress bar).
I'll working on a fix for it, will tell you if i can find one.

**Comment by Albii** [comment]
I've been able to work around this issue by commenting Lines 5494 to 5507 in video.js

```
 // add child elements to represent the individual buffered time ranges
 /*for (var i = 0; i < buffered.length; i++) {
      var start = buffered.start(i);
      var end = buffered.end(i);
      var part = children[i];

      if (!part) {
        part = this.el_.appendChild(Dom.createEl());
      }

      // set the percent based on the width of the progress bar (bufferedEnd)
      part.style.left = percentify(start, bufferedEnd);
      part.style.width = percentify(end - start, bufferedEnd);
    }*/

```
Child elements are added (i counted up to 5.000+ empty div for a 10 min long youtube video) but not correctly removed. Not sure if this part of code is really used and should probably be removed from the main version of video.js as the only thing this method (which is LoadProgressBar.prototype.update) should do is update the progress bar's percentage. 

Anyway this fix will do it for me :)

**Comment by gkatsev** [comment]
I'm guessing this is some weird interaction between videojs and youtube where we're adding a buffered indicator fo each buffered range.

**Comment by gkatsev** [comment]
I belive this is fixed by #411.

**Comment by gkatsev** [comment]
Available in 2.2.0.
